src/GitVersionCore.Tests/BuildServers/VsoAgentTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using GitVersion;
+using NUnit.Framework;
+
+[TestFixture]
+public class VsoAgentTests
+{
+    [Test]
+    public void Develop_branch()
+    {
+        var versionBuilder = new VsoAgent();
+        var vsVersion = versionBuilder.GenerateSetVersionMessage("0.0.0-Unstable4");
+        //  Assert.AreEqual("##vso[task.setvariable variable=GitBuildNumber;]0.0.0-Unstable4", vsVersion);
+
+        Assert.Null(vsVersion);
```

Comment:
Minor thing, but using Shouldly will give much better error messages. `vsVersion.ShouldBe(null)`

I think I am going to move to xUnit so there is not the option for using the inbuilt assertions, not going to block PR though, I will cleanup later